src/Calculator/BcMathCalculator.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * This file is part of the Money library
+ *
+ * Copyright (c) 2011-2013 Mathias Verraes
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Money\Calculator;
+
+use InvalidArgumentException;
```

Comment:
Classes from root namespace are OK to be used in the form of `\InvalidArgumentException`. Import statements like this does not make things easier to use/read.